src/main/kotlin/io/github/slaxnetwork/game/GameManager.kt

```
@@ -125,6 +129,16 @@ class GameManager(
         kotcPlayer?.crownHolder = true
     }
 
+    /**
+     * Update the world's world border.
+     */
+    private fun updateWorldBorder(center: Location, size: Double) {
+        val world = Bukkit.getWorld("world")
```

Can we please throw a NPE here instead of doing null checks?